client/log_client_test.go

```diff
@@ -90,12 +97,17 @@ func TestListByIndex(t *testing.T) {
 		t.Fatal(err)
 	}
 	defer env.Close()
-	logID, err := env.CreateLog()
+	tree, err := CreateAndInitTree(ctx, &trillian.CreateTreeRequest{
```

Comment:
Can you repeat the same indent and keep-together changes in other places in this file as you did above? ( keeping `&trillian.CreateTreeRequest{Tree: stestonly.LogTree}` online line and reflowing around it)

Reply:
yep. Done